src/Client.js

```diff
@@ -99,7 +99,7 @@ export class Client {
 					}
 
 					// resolve when a result is received
-					if (response && response.result) {
+					if (response && typeof response.result !== 'undefined') {
```

Comment:
the original if statement will actually properly filter if the `response.result` is truly `undefined` and this statement check is more specific and feels unnecessary in this case... was this a change for testing purposes that can be backed out?